chore: v1.0.0-beta.79 (#505)
Redocly/openapi-cli: docs/changelog.md

# OpenAPI CLI changelog

## 1.0.0-beta.79 (2022-01-10)

### Fixes

> - Fixed an issue with `remove-x-internal` decorator doesn't remove references to `x-internal` components.

Reviewer: ```suggestion
- Fixed the `remove-x-internal` decorator to remove references to removed `x-internal` components.
```

Author: done
